vendors/nordic/boards/nrf52840-dk/CMakeLists.txt

```diff
@@ -68,25 +80,79 @@ set(
 
 target_compile_definitions(
     AFR::compiler::mcu_port
-    INTERFACE $<$<NOT:$<COMPILE_LANGUAGE:ASM>>:${defined_symbols}>
-)
-#"$<TARGET_FILE_DIR:${exe_target}>  Debug/Obj/${exe_target}.ld"
-set(c_flags "-fmessage-length=0" "-fno-diagnostics-show-caret" "-mcpu=cortex-m4" "-mlittle-endian" "-mfloat-abi=hard" "-mfpu=fpv4-sp-d16" "-mthumb" "-mtp=soft" "-munaligned-access" "-nostdinc" "-quiet" "-std=gnu99" "-gdwarf-4" "-g3" "-gpubnames" "-fomit-frame-pointer" "-fno-dwarf2-cfi-asm" "-fno-builtin" "-ffunction-sections" "-fdata-sections" "-fshort-enums" "-fno-common")
-set(linker_flags "-X" "--omagic" "-eReset_Handler" "--defsym=__vfprintf=__vfprintf_long_long" "--defsym=__vfscanf=__vfscanf_long_long" "-EL"
-"-T${CMAKE_BINARY_DIR}/aws_tests.ld"
-"-Map Output/Debug/Exe/aws_tests.map"
-"-u_vectors"
-"-o Output/Debug/Exe/aws_tests.elf"
-"--emit-relocs"
---start-group "@/home/ANT.AMAZON.COM/hbouvier/Desktop/afr-dev/amazon-freertos/vendors/nordic/boards/nrf52840-dk/aws_tests/ses/Output/aws_tests Debug/Obj/aws_tests.ind" --end-group
+    INTERFACE
+    ${defined_symbols}
 )
 
+set(c_flags
```

Comment:
I'm not sure whether in our CMake structure, there's a way to pass in optimization level flag. If no, I would explicitly call out "no optimization". This way user at least knows where to modify if they want to change optimization level.

Reply:
Good point. Added optimization level to 0.